src/pipecat/audio/turn/smart_turn.py

```diff
@@ -25,9 +24,10 @@ def __init__(self, url: str, **kwargs):
             logger.error("remote_smart_turn_url is not set.")
             raise Exception("remote_smart_turn_url must be provided.")
 
-        # Use a session to reuse connections (keep-alive)
-        self.session = requests.Session()
-        self.session.headers.update({"Connection": "keep-alive"})
+        self.client = httpx.AsyncClient(
+            headers={"Connection": "keep-alive"},
+            timeout=httpx.Timeout(self._params.stop_secs),
+        )
```

Comment:
we should use `aiohttp` which is what we use everywhere.

Reply:
and we should pass an `aiohttp_session` argument as well, as we do with all the services.